packages/babel-helpers/src/helpers.js

```diff
@@ -432,13 +432,29 @@ helpers.objectDestructuringEmpty = template(`
 `);
 
 helpers.objectWithoutProperties = template(`
-  (function (obj, keys) {
+  (function (source, excluded) {
+    if (source === null) return {};
```

Comment:
Nit: actually, I don't think we need to handle `source === null` at all? What's the spec say?